index.html

```diff
@@ -5081,106 +5081,104 @@ <h3>Signature-Based Correlation</h3>
 
         <p>
 The contents of a [=verifiable credential=] are secured using a [=securing
-mechanism=]. Values used to represent the securing mechanism create a greater
-risk of correlation when the same values are used across more than one session
-or domain and the value does not change. Examples of these sorts of values
-include the following:
+mechanism=]. Values representing the securing mechanism pose a greater
+risk of correlation when they remain the same across multiple sessions
+or domains. Examples of these include the following values:
         </p>
 
         <ul>
           <li>
-the binary value of the digital signature,
+the binary value of the digital signature
           </li>
           <li>
-timestamp information associated with when the digital signature was created,
-and
+timestamp information associated with the creation of the digital signature
           </li>
           <li>
-cryptographic material, such as a public key identifier, associated with the
+cryptographic material associated with the digital signature, such as
+a public key identifier
 digital signature.
```

Comment:
```suggestion
cryptographic material associated with the digital signature, such as
a public key identifier.
```

Reply:
Minus the terminating period. Punctuation is generally not appropriate for bullets, especially not for *one* bullet in the set.